src/api/app/validators/workflow_steps_validator.rb

```diff
@@ -30,14 +29,20 @@ def no_steps?
 
   def errors
     acc = []
```

Comment:
what is `acc`? couldn't we use a better name here?

Reply:
definitely!

Reply:
done